src/lib/coda_networking/coda_networking.ml

```diff
@@ -406,6 +453,10 @@ module Make (Inputs : Inputs_intf) = struct
 
   type transaction_pool_diff = Inputs.Transaction_pool_diff.t
 
+  type ban_notification = Gossip_net.ban_notification =
+    {banned_peer: Peer.t; banned_until: Time.t}
```

Comment:
Should we include the ban reason in here too?

Reply:
The trust system doesn't really support that, currently. @o1pranay was going to add an issue so that could be done. I'd say that's for another PR.

Reply:
Issue is here: #3062